smartsnippets/tests/tests.py

```diff
@@ -155,6 +160,29 @@ def test_vars_generated(self):
         self.assertEqual(Variable.objects.count(), 3)
         self.assertEqual(plugin3.variables.count(), 1)
 
+    def test_validation_passes_for_correct_variables(self):
```

Comment:
Can you add some tests with valid variable names? Just to make sure that the base functionality is not broken.